unity-renderer/Assets/DCLServices/MapRendererV2/MapLayers/Atlas/ChunkAtlasController.cs

```diff
@@ -9,26 +9,21 @@ namespace DCLServices.MapRendererV2.MapLayers.Atlas
 {
     internal class ChunkAtlasController : MapLayerControllerBase, IAtlasController
     {
-        public delegate UniTask<IChunkController> ChunkBuilder(Vector3 chunkLocalPosition,
-            Vector2Int coordsCenter,
-            Transform parent,
-            CancellationToken ct);
+        public delegate UniTask<IChunkController> ChunkBuilder(Vector3 chunkLocalPosition, Vector2Int coordsCenter, Transform parent, CancellationToken ct);
 
         public const int CHUNKS_CREATED_PER_BATCH = 10;
```

Comment:
I think you should lower this number for both this class and your new one as it will be executed in parallel and cause additional hiccups than expected initially. Consider reducing it by twice (as you have 2 implementations)

Reply:
I would not load it on demand as we have the background capacity to do it. And you count it as mandatory as it's more readable than Sattelite. How about to start loading it after Sattelite then (not in parallel)?